cockpit/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "a plugin for cockpit to manage subscriptions",
   "main": "index.js",
-  "repository": "git@github.com:rarebreed/subscription-manager.git",
```

Comment:
We should consider filling out this section fully, like where to open bugs and all the other project metadata...

Also everything in this file is listed as a devDependency , (previously added with `yarn add package --dev` from the README), this should really be reserved for test deps and transpilers and things not needed to actually use the project.

This overhaul would be a good item to make a card about.